diff -Nuard xrdesktop-0.5.orig/xrdesktop xrdesktop-0.5/xrdesktop
--- xrdesktop-0.5/xrdesktop.orig	Tue Feb 11 21:32:00 2003
+++ xrdesktop-0.5/xrdesktop	Tue Feb 11 21:33:59 2003
@@ -63,11 +63,11 @@
 my $selected_machine_ip;
 my $selected_uname;
 my $selected_geom;
-my $selected_fs;
-my $selected_bitmap_upd;
+my $selected_fs="";
+my $selected_bitmap_upd="";
 my $selected_kmap;
 my $selected_kmap_hex;
-my $selected_lic;
+my $selected_lic="";
 my $dialog1;
 my $dialog_button_ok;
 my $dialog_button_cancel;
